<a href="https://colab.research.google.com/github/nileshdgaikwad8805/Data-Science-Assignments/blob/main/Q9_b.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:

import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
%matplotlib inline

In [8]:
data = pd.read_csv("/content/Q9_b.csv")
data.head()

Unnamed: 0          SP         WT
0           1  104.185353  28.762059
1           2  105.461264  30.466833
2           3  105.461264  30.193597
3           4  113.461264  30.632114
4           5  104.461264  29.889149

In [6]:
new_data = data.iloc[:,1:]

In [7]:
new_data

SP         WT
0   104.185353  28.762059
1   105.461264  30.466833
2   105.461264  30.193597
3   113.461264  30.632114
4   104.461264  29.889149
..         ...        ...
76  169.598513  16.132947
77  150.576579  37.923113
78  151.598513  15.769625
79  167.944460  39.423099
80  139.840817  34.948615

[81 rows x 2 columns]

In [9]:
#skewness
new_data.skew()

SP    1.611450
WT   -0.614753
dtype: float64

In [10]:
#kurtosis
new_data.kurt()

SP    2.977329
WT    0.950291
dtype: float64

Skewness:

For the attribute SP, the skewness is 1.611450. A positive skewness indicates that the distribution of SP is skewed to the right, meaning that it has a longer right tail.
For the attribute WT, the skewness is -0.614753. A negative skewness indicates that the distribution of WT is skewed to the left, meaning that it has a longer left tail.

